erpnext/patches/v13_0/create_leave_policy_assignment_based_on_employee_current_leave_policy.py

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) 2019, Frappe and Contributors
+# License: GNU General Public License v3. See license.txt
+
+from __future__ import unicode_literals
+
+import frappe
+
+def execute():
+    employees_with_leave_policy = frappe.db.sql("SELECT name, leave_policy FROM `tabEmployee` WHERE leave_policy IS NOT NULL", as_dict = 1)
+
+    for employee in employees_with_leave_policy:
+        print(employee.name)
+        if not frappe.db.exists("Leave Allocation", {"employee":employee.name, "leave_policy": employee.leave_policy, "docstatus": 1}):
```

Comment:
Leave policy assignment should be created even if there are leave allocations.

If leave allocation record is linked with leave_policy, then leave_policy_assignment should also be updated

Reply:
done